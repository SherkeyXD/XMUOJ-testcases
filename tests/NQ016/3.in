-229A2690 -3B10616C
-35AEED40 -3795CB80
-33A8DA08 -37315B20
-3B0108B8 -338AFE06
-27C9D762 -39A81678
-28D484EB -34140A60
-3B252293 -39BD92F8
-2C3B7140 -2B74E5D4
-39E6AFD9 -1DA3F214
-32256368 -2A8700A3
-3979EAB1 -2F288CEC
-1B6727C4 -31A46988
-9AAC840 -1B94A88C
-1EEB56FD -1C14F444
-353AE1F3 -3B389058
-C491E60 -20334B3E
-33F661F0 -232BF160
-382B0C16 -2DEAE4C1
-215146F6 -34A2B46A
-3180F9B6 -39E852FC
-276A36DE -21BAF68B
-27339E20 -33D6731A
-3589BBD0 -3AF84E94
-398F9B60 -356D2CE0
-2DB2EDC0 -36C352F8
-39BFD208 -175E6D2C
-339000F1 -385E4CB3
-13C17754 -2640C5B0
-35C2EBF4 -3A7A2152
-2C7BC9AD -D11D670
-24105A12 -10DDEFA0
-3B733688 -3190CA36
-305D23A -1E2BD182
-364281E5 -2688C006
-3535B9F8 -303934E4
-3B46F6D6 -2766B1AA
-37ADCD7A -18CA95AC
-25BD6CE4 -33F86380
-866A55A -25C3806C
-32367B5D -3A0183BA
-231C9A10 -3A6AA3D0
-166B54BC -2E8474F1
-2CB5D03C -25223280
-3A2D3E5C -19D87168
-1D029EAC -23156609
-7D5BA67 -276FB890
-35A74AE2 -109F70F8
-148F9F87 -B208F77
-33D0C387 -24FCCBBC
-2C39AC43 -2DB37348
-24501871 -1880350E
-19EA6D70 -32480AA8
-3ABF40E7 -3B80264D
-3050C596 -2FD07D3D
-348D14EC -17797959
-1F2D4164 -39D22F08
-37BD1F30 -2B31A1C6
-1C918A02 -3A127E3F
-3813FE02 -35416880
-3448D93C -3AFEBCF8
-773D09A -1709B018
-292A3EAA -17EF3EB5
-39A13DCE -338FB5E4
-3A302187 -2AA4B8AA
-30E0DB5A -396394F2
-1D7E2600 -34416AED
-1733BE31 -34D5D962
-20BFBB14 -1933E0EE
-395126BF -39B4481C
-255A73B2 -3B301B56
-3142024E -B4C1E98
-301432C4 -37F96844
-85DEA7F -33F1B520
-31235C68 -38B7C014
-2224E534 -31842528
-3A670EE0 -19930EAC
-3A062547 -34413E38
-3B8D6FA0 -2B249A7F
-3930ABB8 -170C89E4
-310EF5C9 -342BCD28
-33BC9E12 -17D71534
-382E6102 -192F5CB6
-38B0DAE7 -28DF650B
-38BEA98C -251AC112
-3934C534 -94EB174
-3B5A3B9C -F336010
-38AF2554 -F7057A6
-194264FE -37D47EC6
-35ED965F -31821028
-349A1EDD -2F4C4C6A
-31D53F5A -28B1CCB2
-3966C93C -12E32CAB
-365B0E38 -30A217B0
-1E8D75D5 -1498FA0E
-10710792 -3746C0C8
-27C5E000 -C159551
-1F5EA41A -255985B5
-34AF5CD2 -30C3FB5D
-343BA126 -39BD6D28
-2B26AC4E -382CCAB6
-21A4E595 -3984D650
-37779AA3 -39AC75F3
-1F9A3A70 -2AE9F4A0
-37C165EC -38BE697D
-3B78BE84 -22DD9850
-1C5B28C4 -3584C968
-2F44FEE2 -1A4B9530
-36859CF2 -353DA714
-28141F3E -338DEA04
-2CE03FAC -3980F02C
-1A320777 -2ADA90E9
-3B1653BC -2F0ED09C
-3179B044 -2DDBC79D
-8B64016 -2D34B3FA
-29EDE622 -389C3CA8
-3599B5E0 -231E7C10
-2A0F6068 -3B578087
-26D0C7E3 -FDE1FE4
-1F18B454 -12B41200
-34004394 -1CDD3288
-321F6088 -38A4A7E8
-265F65E0 -1A0D18DB
-362FA6B4 -23CE9F46
-1712549E -74E59FA
-39F2413C -2660366C
-37E79B3E -1DB5293B
-27107538 -370CEF36
-97CBE44 -3B2C39D4
-39BF0074 -3786067A
-3A8303AC -2599710A
-39672812 -30107AE0
-340A8F02 -348A3E28
-32672B58 -39EF1400
-29E7A1A9 -3B7EFCC4
-2F36767A -29AFF45A
-2E32E4B0 -2E70EE80
-39A34E2A -3B45FF4D
-289A598C -27DB937C
-221AEC80 -2BE169DE
-211C89CF -3A769DFD
-FB009E3 -272A603F
-3AE6B1B1 -31932B33
-2E33E9C0 -37ECBE8B
-3914DCEB -360AFC9C
-2DE12908 -35BB7870
-319588D8 -3289CD9E
-1D4A82D9 -29F8985F
-3B6FDCBE -349211EC
-38FC84A5 -1E0FEF88
-1D6121B8 -23F9D906
-37E47EB0 -3A094F41
-36CA35D6 -338F97A0
-918BC95 -33C34168
-3B4BFA30 -15080777
-32BAC29C -10CA0549
-310CBEE6 -2064C1B4
-39DEB86A -1752C7A6
-14EEF0A0 -1D8BC3FD
-397BDC8C -21801208
-320A7AAD -29601704
-3A3A3C36 -2C98AA80
-33271780 -27EE8A00
-34339123 -26EF2877
-3A386D50 -3A3872AC
-25660E4F -34CFB956
-1AD71E89 -33B2F9E6
-32B619C0 -31BE88EB
-34A99F80 -38248954
-21BFA600 -30A44CF7
-4AA9850 -18F37672
-3B50046B -2377B87B
-3797F28B -3202F9E5
-297F43D7 -3B92F21F
-380492B4 -EE552B8
-1DF8C992 -305F2750
-1A89177C -18B9CDC0
-3B76DEB8 -32AE14D0
-77B9042 -39DD293B
-366737EE -1EF490BA
-2D508F00 -249D7080
-372E60E6 -3AB8D430
-38D6A3E0 -3029F852
-269EF3D8 -3677FE6A
-39F2A8F6 -2E3504F4
-1EC5D272 -34471EB0
-393A337B -38B8C0DF
-17F9CBD0 -39499E40
-156BCA73 -3821B012
-36FCC6B2 -99A9B60
-308A47EE -26CB2034
-3B9161F6 -12BCCF5C
-37173275 -35C723F4
-2304133A -323CF9E2
-3518649C -21A14C33
-3A61146F -398308F5
-EE4EEFA -3A30DFEC
-36B22DCE -17C74180
-385473CC -1341BC6D
-1E2218B5 -3A6EE233
-1DAD814C -2B4D1D4B
-38A03024 -346A0C50
-1B79BE03 -2C24FB54
-11E737E4 -110A975F
-2111B851 -2A4E341A
-2A7D72EA -1B030BC2
-12C555AE -1B830864
-34DE352E -346A7ABC
-36E114A4 -B67B47
-31751390 -29CF61E3
-2471149A -102EB43A
-320FE07A -39679744
-3A33784C -2E6734CA
-305E154E -35A50390
-29B87371 -29DAD29C
-3985134A -2F199328
-28AE11D8 -2EB6E810
-28BFB198 -2A2B5646
-115D8E91 -23F2F6BB
-29B3E637 -1B58E5C7
-36757AAE -2B49CF5C
-1DE2F64A -301B997C
-1F0CAE1C -334EC28A
-3A4CA50F -36830B90
-2FE1E076 -325B9906
-2A259A09 -39BA5E0C
-2A81FF34 -2F56AA28
-32372B3C -3B472A25
-333EB164 -217A95D4
-2CAF9670 -18F70FE0
-30E74576 -3AA7E67D
-34A2879F -B256888
-2FADFB3A -3AC7854C
-11A00CE0 -2E19143E
-37143AC0 -290E1EEE
-2B2129CC -3930A9AC
-38C1AB46 -23A2080C
-3A1A15E8 -3B3D7918
-26082EBC -3ACA2FA8
-2E4EC1C4 -1DEF0960
-31CE49DC -1FED0E46
-1056CDAE -21496ACE
-6AE6F60 -35D858C8
-3B4CB961 -1C9949B9
-32C4AAE0 -35B3F948
-1C6FB83C -39FB4AB4
-32E2F7FC -3AB69291
-25493368 -33395877
-3122D24C -3A826E97
-2721FFD3 -3A5296C0
-24250564 -7E81DFC
-2B5CB9CC -34BF1970
-74F8B7A -272FC724
-3475BBFC -2B510630
-8218B7C -15F1C83A
-3A2CD5F0 -24023E96
-B1FD0D2 -188860F3
-E02398C -2E2E3D24
-2FF4B198 -38CB7266
-2AE90DB8 -2455645F
-38881F94 -2F2B1492
-2584099B -37798198
-251EE820 -27FC9604
-3B81BAC0 -379FB050
-3ADC3059 -165A5CF6
-15FE0BC4 -72EC332
-2E6F689F -346229CE
-2ACAE5B4 -3A5F01C0
-27C4A380 -385C2898
-14A7997E -3AB07D01
-2AD15F8C -36F6CFB2
-3712B233 -11C2C870
-93CD4C5 -23A269F8
-33FD2F12 -2FE38799
-2255FB16 -30C5D910
-F5E9F1B -2BB2E2A6
-2DDD7B56 -37621675
-18A844E0 -32EA2AC6
-38968D9E -3161C794
-36A1FDAC -24E38310
-25B6B069 -19136EF0
-3419F76E -30506D44
-39CA270C -368F88D4
-15BDEEE8 -17F8228C
-3398FB2E -3B2D01C8
-34D08F5D -34EFE9F4
-32FFCE80 -2779614A
-37FECEA8 -1292D5B0
-25980786 -1A5F77C4
-2BDEC45D -3AFECF98
-3B54553A -321D0E73
-2E726D65 -3E656F0
-38D1CA32 -1B9BA496
-1859973D -4B66928
-220C6B67 -C903660
-26ED69BC -3B1F0840
-27F3FB20 -38A89830
-35BD2C77 -3979C84C
-22AD4090 -37EA8844
-39347650 -3590566D
-35DF6490 -3A05F3F8
-3533D598 -8E8AFFC
-8EE9FEE 1404608
-1D7830D0 -1840AA7A
-2AA41D64 -324484AE
-3B23CD9E -3B0EA5DC
-140B0376 -33CC6EE6
-36FAF9FC -36DCFB28
-743D5B0 -27A76BD8
-1F6F40BE -2BBE081E
-2F442134 -3745D76C
-2D33C914 -28808A86
-34CD4907 -2E611D16
-32D1E4D0 -2F0734D7
-260FE1C0 -36A188A0
-379643DA -212E07B0
-30C83400 -378B40EE
-3A46CD2F -DB1E484
-2F911234 -1D1428A0
-3694BB24 -1A5F2568
-35ECACE0 -3423B656
-37827A51 -2736978A
-3765A67B -1CCC7480
-39646444 -2F01F9AC
-39773F88 -38A72E92
-28E8915C -296EA2BC
-34B26FE0 -1855EBB3
-38A30799 -22BFDB4C
-291520DA -1DC0F8F0
-297B4EC6 -3B073166
-3B8F6722 -330C0CA0
-3A41B190 -26C1C9B0
-206D10A8 -26DF4BBE
-63785C5 -38A99580
-3B097F60 -31EA8079
-2B3FC906 -2D59B60E
-D970638 -1EED256E
-3A780B16 -2C32D95C
-27675445 -36413C38
-18087E65 -33995B06
-36927F1C -274239C5
-26C424B0 -27772C6F
-38F96662 -35D95944
-362DAB32 -321F6770
-316B7F4E -25FC348C
-17E352BC -30342006
-30DE2E36 -BE3799E
-28AA1BB8 -3B5279B2
-37914236 -1488B4B3
-38F53F78 -3B70E455
-261E568A -12AEAE40
-3156F2E0 -2B5298D8
-348F355B -421F660
-3B3C07F1 -2309626A
-39E9C760 -2C9A92B0
-3A4B9314 -26E0EBCE
-38EEF050 -36DE562F
-1B70379B -392C7D66
-316EF349 -2DF8732E
-33ED9ABF -2CCA35F6
-3A5A3723 -2EA87D10
-3B5E22A1 -2F01DCDC
-3A577620 -33580ED0
-2D40232E -33B00772
-21C8BF80 -3037FAB3
-1C07613E -35B96C80
-32087338 -392C22B8
-3B7A6C47 1CBA242
-2B6BB20C -194AFC83
-18C46D20 -131458E4
-387CD87E -38BB756E
-3553E960 -15B5EEDA
-2FA2F5C0 -3B1561C0
-2C8DDFFC -395808C5
-20A63408 -15FC9506
-3B55010C -29D0E5EA
-1CAE66C2 -358643A5
-2994964C -37A0F433
-23A898E8 -3583FBE8
-2EF1F6EA -389FEDBC
-30372C1C -24D65014
-38EFCCE0 -15A1D6BC
-27204F32 -20E802DD
-3B884D12 -2D3F44E0
-3B01A6C6 -3B3544AA
-19C40456 -329A53E6
-260DB8D8 -30130681
-21D7F63C -30D8C60A
-2DC100CF -3B310C58
-1ABFCC86 -3148A678
-16C690C4 -37F4E7C0
-35F894F8 -24F3D129
-2F45E61B -1CBC7CC0
-32DCA354 -32E2518C
-1E8549FC -36C8898D
-26D54DC0 -126EAFCC
-33C4B437 -36167886
-37E1C0C1 -3B47370A
-33194D72 -35910418
-384CDEA5 -37FF581E
-3B22C74C -2A27A040
-337E9DF0 -22D97336
-20AFAA57 -27D2E350
-29F0DD12 -2C2B3404
-3595B6FE -37290A2D
-16CF6D68 -16E7E1C0
-135987F6 -30E507E2
-486931C -322A52B5
-35504B60 -2CE5EDA6
-2D3CD13E -2FA4FFF0
-25C36372 -2AC7D2A8
-33ECDEAC -32C15338
-1AD19FCC -2FD22CA5
-24CD7BCC -38414647
-D87FB0C -31A84EC3
-3086B390 -37797D98
-3AC32478 -31BBDE38
-3B21C5F5 -C033230
-34FC9B7C -2D3DBFB5
-9418FA -28104C24
-32159EED -1644D840
