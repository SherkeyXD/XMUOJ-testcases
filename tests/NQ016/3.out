-5DAA87FC
-6D44B8C0
-6ADA3528
-6E8C06BE
-6171EDDA
-5CE88F4B
-74E2B58B
-57B05714
-578AA1ED
-5CAC640B
-68A2779D
-4D0B914C
-253F70CC
-3B004B41
-7073724B
-2C7C699E
-57225350
-6615F0D7
-55F3FB60
-6B694CB2
-49252D69
-5B0A113A
-70820A64
-6EFCC840
-647640B8
-511E3F34
-6BEE4DA4
-3A023D04
-703D0D46
-398DA01D
-34EE49B2
-6D0400BE
-2131A3BC
-5CCB41EB
-656EEEDC
-62ADA880
-50786326
-59B5D064
-2E2A25C6
-6C37FF17
-5D873DE0
-44EFC9AD
-51D802BC
-5405AFC4
-401804B5
-2F4572F7
-4646BBDA
-1FB02EFE
-58CD8F43
-59ED1F8B
-3CD04D7F
-4C327818
-763F6734
-602142D3
-4C068E45
-58FF706C
-62EEC0F6
-56A40841
-6D556682
-6F479634
-1E7D80B2
-41197D5F
-6D30F3B2
-64D4DA31
-6A44704C
-51BF90ED
-4C099793
-39F39C02
-73056EDB
-608A8F08
-3C8E20E6
-680D9B08
-3C4F9F9F
-69DB1C7C
-53A90A5C
-53FA1D8C
-6E47637F
-66B20A1F
-503D359C
-653AC2F1
-4B93B346
-515DBDB8
-61903FF2
-5DD96A9E
-428376A8
-4A8D9BAC
-481F7CFA
-5116E3C4
-676FA687
-63E66B47
-5A870C0C
-4C49F5E7
-66FD25E8
-33266FE3
-47B7C85A
-33DB7551
-44B829CF
-6573582F
-6DF90E4E
-63537704
-5B29BBE5
-71241096
-4A842F10
-707FCF69
-5E5656D4
-51DFF22C
-49909412
-6BC34406
-5BA20942
-66612FD8
-450C9860
-6A252458
-5F5577E1
-35EAF410
-628A22CA
-58B831F0
-6566E0EF
-36AEE7C7
-31CCC654
-50DD761C
-6AC40870
-406C7EBB
-59FE45FA
-1E60AE98
-605277A8
-559CC479
-5E1D646E
-44A8F818
-714506EE
-601C74B6
-6977A2F2
-6894CD2A
-6C563F58
-65669E6D
-58E66AD4
-5CA3D330
-74E94D77
-5075ED08
-4DFC565E
-5B9327CC
-36DA6A22
-6C79DCE4
-6620A84B
-6F1FD987
-639CA178
-641F5676
-47431B38
-7001EEAA
-570C742D
-415AFABE
-71EDCDF1
-6A59CD76
-3CDBFDFD
-505401A7
-4384C7E5
-5171809A
-51318010
-327AB49D
-5AFBEE94
-5B6A91B1
-66D2E6B6
-5B15A180
-5B22B99A
-7470DFFC
-5A35C7A5
-4E8A186F
-6474A2AB
-6CCE28D4
-5263F2F7
-1D9E0EC2
-5EC7BCE6
-699AEC70
-651235F6
-46E9E56C
-4E57F0E2
-3342E53C
-6E24F388
-4158B97D
-555BC8A8
-51EDFF80
-71E73516
-69009C32
-5D16F242
-6827ADEA
-530CF122
-71F2F45A
-51436A10
-4D8D7A85
-40976212
-57556822
-4E4E3152
-6CDE5669
-55410D1C
-56B9B0CF
-73E41D64
-4915CEE6
-4E796F4E
-4B963039
-5890FAE8
-48FA9E97
-6D0A3C74
-479EB957
-22F1CF43
-4B5FEC6B
-45807EAC
-2E485E12
-6948AFEA
-37978FEB
-5B447573
-349FC8D4
-6B7777BE
-689AAD16
-660318DE
-5393460D
-689EA672
-5764F9E8
-52EB07DE
-3550854C
-450CCBFE
-61BF4A0A
-4DFE8FC6
-525B70A6
-70CFB09F
-623D797C
-63DFF815
-59D8A95C
-6D7E5561
-54B94738
-45A6A650
-6B8F2BF3
-3FC7F027
-6A758086
-3FB9211E
-602259AE
-6451D378
-5C63B352
-75578F00
-60D25E64
-4C3DCB24
-51BB5822
-31A0387C
-3C86C828
-57E6031A
-6878A428
-566B02F0
-6D998A8D
-58828BDF
-6BA540E3
-61749693
-2C0D2360
-601BD33C
-2E7F529E
-5FC6C22C
-1E1353B6
-5E2F1486
-23A831C5
-3C3076B0
-68C023FE
-4F3E7217
-67B33426
-5CFD8B33
-4D1B7E24
-73216B10
-51368D4F
-1D2CCEF6
-62D1926D
-6529E774
-6020CC18
-4F58167F
-61C82F3E
-48D57AA3
-2CDF3EBD
-63E0B6AB
-531BD426
-3B1181C1
-653F91CB
-4B926FA6
-69F85532
-5B8580BC
-3ECA1F59
-646A64B2
-7059AFE0
-2DB61174
-6EC5FCF6
-69C07951
-5A792FCA
-4A91A458
-3FF77F4A
-66DD93F5
-6D7163AD
-3258C455
-546D6EC8
-1D100065
-2E9CA1C7
-620C71FC
-609C9350
-6F36F4C3
-5A97C8D4
-6EC4CCBD
-6FE55888
-3E1C8594
-7AE59E6
-35B8DB4A
-5CE8A212
-7632737A
-47D7725C
-6DD7F524
-2EEB4188
-4B2D48DC
-6689F8A0
-55B4539A
-632E661D
-61D919A7
-5CB16A60
-58C44B8A
-685374EE
-47F8B1B3
-4CA53AD4
-50F3E08C
-6A106336
-5EB911DB
-54321AFB
-68665DF0
-721E6E1A
-52573418
-4D085B93
-5B62E2E5
-46D619CA
-6482802C
-6E9B73C2
-61037B40
-474C5C66
-3EE11B45
-6CF3FFD9
-58997F14
-2C842BA6
-66AAE472
-5DA8907D
-4BA1D96B
-5DD4B8E1
-4E3B511F
-6ED2BFA6
-684D12A2
-5767B3DA
-481772C2
-3CC1A7D4
-63FC956A
-4C19F6E9
-746623CD
-38CD04CA
-5CA98BB8
-38B12BBB
-5E456A5B
-66845A10
-612C7EE2
-6FCD467F
-549CB501
-5F676677
-60B7D0B5
-6902B433
-6A5FFF7D
-6DAF84F0
-60F02AA0
-5200BA33
-51C0CDBE
-6B3495F0
-39AECA05
-44B6AE8F
-2BD8C604
-71384DEC
-4B09D83A
-6AB85780
-65E5E8C1
-36A2C90E
-6525E6F6
-5234AA67
-61358A7F
-592C94D0
-6791E4A6
-550D7C30
-4E91A39C
-4808520F
-68C791F2
-7636EB70
-4C5E583C
-5620BF59
-52B0BC46
-68F20D27
-4C0872FE
-4EBB7884
-5AEC6621
-4C0262DB
-65BEF4E0
-554DD389
-3943FD8C
-69DB2CBD
-7328F7CB
-68AA518A
-704C36C3
-654A678C
-56581126
-48828DA7
-561C1116
-6CBEC12B
-2DB74F28
-443E8FD8
-36B0E5D1
-62363906
-5CE1D12E
-508B361A
-66AE31E4
-4AA3CC71
-5D0EC213
-3F3049CF
-68003128
-6C7F02B0
-4724F825
-623A5B31
-28A4651E
-485A772D
